CHANGELOG.rst

```diff
@@ -2,18 +2,22 @@
 
 0.19.2
 ------
-This minor bugfix release comes with internal changes to the Knowledge Graph and a small fix for the Lineage API.
+This minor bugfix release comes with internal changes to the Knowledge Graph and a bug fix on the core service.
 
 Bug fixes
 ~~~~~~~~~~
 
 * **Knowledge Graph**: Lineage API to read data from the new dataset in the Triples Store
+* **Knowledge Graph**: migration fixing creation dates on Plans in the Triples Store
+* **Renku core service**: a bug causing Plan creation date to be after Activity start date
```

Comment:
Should we always start these sentences with a verb? I'm not sure we have a fixed rule for that, but that's what we tired to do lately. Feel free to reword any of the suggested sentences

```suggestion
* **Knowledge Graph**: read lineage data from the new dataset in the Triples Store
* **Renku core service**: prevent Plans creation dates from preceding their corresponding Activities
* **Knowledge Graph**: fix wrong Plans creation dates through migration
```